.eslintignore

```diff
@@ -4,3 +4,4 @@ node_modules
 website
 **/*.json
 **/*.md
+**/lib
```

Comment:
Why do we have this directory? This seems to be a duplicate of `dist/esm/`.

Reply:
Before `api-extractor` can run, `tsc` has to run to build the regular declaration files, they are used to generate the rollup declaration. So, there are definitely some things that could be reconsidered. Since `tsc` has to be run to generate declarations, it could also be used to generate the es modules. I've been using it with the build flag `-b` in some other projects, along with tsconfig project references and it only builds files that change. It is reasonably quick. Another benefit of project references and incremental build is that the tsconfig files are links through references. That means that the types don't have to be rebuilt for them to be correct in VSCode. If you've noticed, with the current configuration, if types change in one project they are wrong until they are rebuilt. With project references, the source code is used directly rather than the packages typings file. I could incorporate some of those changes in a different PR for you to take a look at, if you're interested.

Reply:
Since they will be rolled up, they just need to be placed in a temporary location. I used `lib` to just follow TypeScript convention.